src/lib/actions/click-outside.ts

```diff
@@ -0,0 +1,16 @@
+/** Dispatch event on click outside of node */
+export function clickOutside(node: any) {
```

Comment:
`node` here is actually a `HTMLElement`

```suggestion
export function clickOutside(node: HTMLElement) {
```